src/TagInput/tests.jsx

```diff
@@ -80,4 +80,90 @@ describe( 'TagInput', () =>
                 .to.be.true;
         } );
     } );
+
+    describe( 'disabled state', () =>
+    {
+        beforeEach( () =>
+        {
+            wrapper.setProps( { isDisabled: true } );
+        } );
+
+        it( 'input should receive isDisabled', () =>
```

Comment:
very minor correction: it should be `input should receive isDisabled as "disabled"`